akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2ServerDemux.scala

```diff
@@ -83,7 +88,7 @@ class Http2ServerDemux extends GraphStage[BidiShape[Http2SubStream, FrameEvent,
 
       override protected def logSource: Class[_] = classOf[Http2ServerDemux]
 
-      val multiplexer = createMultiplexer(frameOut, StreamPrioritizer.first())
+      override val multiplexer = createMultiplexer(frameOut, StreamPrioritizer.first())
```

Comment:
I was still operating under the old rule :-)
I don't mind if we explicitly want to use the other rule - should we add it to the contributing guide too then?